mapf_environment/Observation observation
float32 reward
bool done
